SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -16,4 +16,6 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
         this.view = view
         view.setLabel(createApplicationScreenMessage())
     }
+
```

Comment:
would be a cleaner diff for future PRs if you don't commit redundant empty lines :) I can show how to selectively commit lines in android studio/command line if you haven't seen it already?